internal/service/iam/user.go

```diff
@@ -225,6 +264,14 @@ func resourceUserRead(d *schema.ResourceData, meta interface{}) error {
 
 	d.Set("user_id", user.UserId)
 
+	tags := KeyValueTags(user.Tags).IgnoreAWS().IgnoreConfig(ignoreTagsConfig)
+	if err := d.Set("tags", tags.RemoveDefaultConfig(defaultTagsConfig).Map()); err != nil {
+		return fmt.Errorf("error setting tags: %w", err)
+	}
+	if err := d.Set("tags_all", tags.Map()); err != nil {
```

Comment:
```suggestion
	}
	
	if err := d.Set("tags_all", tags.Map()); err != nil {
```

И тут тоже удобнее параметры по алфавиту обрабатывать